backend/app/interactem/app/api/routes/pipelines.py

```diff
@@ -298,3 +301,37 @@ async def run_pipeline(
         raise HTTPException(status_code=500, detail=f"Failed to run pipeline: {e}")
 
     return PipelineRevisionPublic.model_validate(revision)
+
+@router.post(
+    "/{id}/revisions/{revision_id}/stop", response_model=PipelineRevisionPublic
```

Comment:
I am not sure it makes sense to return a pipeline revision here, if we are using 'POST' we should be creating a 'stop operation" or something that can be used to report the success or otherwise of the request to stop?

Reply:
done